src/client/vite.config.ts

```diff
@@ -302,6 +303,32 @@ const setConfig: (env: ConfigEnv) => UserConfigExport = ({ mode }) => {
   plugins.push(htmlPlugin(isDev))
 
   const isStorybook = process.env.STORYBOOK === "true"
+  const fontfacePreLoad = Unfonts({
+    google: {
+      families: [
+        {
+          name: "Lato",
+          defer: false,
+          styles:
+            "ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900",
+        },
+        {
+          name: "Roboto",
+          defer: false,
+          styles:
+            "ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900",
+        },
+        {
+          name: "Montserrat",
+          defer: false,
+          styles:
+            "ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900",
+        },
```

Comment:
possibility that we dont need all these weights, but include them just to make sure, the ones we are sure we don not need can be removed after, if we go with this solution

Reply:
what is the output of this plugin (in the build) ?
i.e. is it in the correct order in plugins array?

Reply:
I take it you have tried production build and `npm run serve`